helm/scylla/values.yaml

```diff
@@ -23,6 +23,11 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+rbac:
+  clusterRole:
+    # Define additional rules to add to the ClusterRole
+    rules: []
```

Comment:
I'm open to discussing alternative naming schemes, I think this would offer the best extensibility for the future.